site/en/docs/privacy-sandbox/permissions-policy/index.md

```diff
@@ -0,0 +1,308 @@
+  Manage how your page and third-party iframes on your page have access to browser features
+date: 2022-03-31
+authors:
+  - kevinkiklee
+---
+
+<!--lint disable no-smart-quotes-->
+
+## Description
+
+There are numerous features available in the browser that provide important functionality to the user. A video camera allows you to chat with people around the world and GPS enables driving in an unfamiliar neighborhood. Unfortunately, these important features can be misused for malicious purposes that violate user privacy. Developers should have a clear understanding of who has access to browser features, and control that access. 
+
+Permissions Policy allows the developer to control the browser features available to a page and its iframes, by declaring a set of policies for the browser to enforce. The policies are applied to origins provided in a response header origin list. The origins in the origin list can be same-origin (where your code resides) or cross-origin (where someone else’s code resides). 
+
+### Changes from Feature Policy
+
+Permissions Policy has been renamed from Feature Policy. The key concepts remain the same between the two, but there are some important changes along with the name: 
+
+#### Structured Fields usage
+
+[Structured Fields](https://www.rfc-editor.org/rfc/rfc8941.html) provide a set of common data structures to standardize parsing and serialization of HTTP header field values. You can learn more about Structured Fields from the [introduction article](https://www.fastly.com/blog/improve-http-structured-headers).
+
+* Before (Feature Policy):
+  * `geolocation ‘self’ https://example.com; camera ‘none’`
+* After (Permissions Policy): 
+  * `geolocation=(self "https://example.com"), camera=()`
+
+#### The header origin list must be combined with the allow attribute in the iframe tag 
+
+* WIth Feature Policy, you were able to enable the feature to a cross-origin frame by either adding the origin to the header origin list or adding an allow attribute to the iframe tag. With Permissions Policy, if you add a cross-origin to the origin list, the iframe tag for that origin must include the allow attribute.
+* If the response does not contain a Permissions Policy header, the origin list is considered to have the default value of `*`, and just adding the allow attribute to the iframe allows access to the feature.
+* Therefore, it is recommended that the Permissions Policy header is explicitly set in the response, so that cross-origin iframes that are not listed in the origin list will be blocked from accessing the feature even with an allow attribute in the tag.
+
+Feature Policy can still be used after Chrome 88, but it acts as an alias for Permissions Policy. Other than the syntax, there is no difference in logic between the two. If both Permissions Policy and Feature Policy are used together, the Permissions Policy header will have higher priority, and will overwrite the value provided from the Feature Policy header. 
```

Comment:
I want to explicitly call out that it's the header here, because Feature Policy can also represent the `document.featurePolicy` object, and the priority part applies to the headers.